ML-Frameworks/pytorch-aarch64/CHANGELOG.md

```diff
@@ -5,15 +5,22 @@ docker/pytorch-aarch64 will be noted in this log.
 Monthly increments are tagged `tensorflow-pytorch-aarch64--rYY.MM`,
 where `YY` is the year, and `MM` the month of the increment.
 
-## [unreleased]
+## [r25.03.1] 2025-03-21
```

Comment:
If there's a point release, I don't think it will be today? Suggest we leave this and line 9 out and update when we're ready for a point release?

Reply:
Done